src/additional_services/prometheus_grafana.star

```diff
@@ -77,6 +77,8 @@ def launch_panoptichain(
         l2_el_genesis_artifact=l2_el_genesis_artifact,
     )
 
+    heimdall_version = 2 if l2_context.devnet_cl_type == "heimdall-v2" else 1
```

Comment:
nit: to make the package more robust - it should fail if the devnet cl type is different from the two options

```suggestion
    heimdall_version_map = {
        constants.CL_TYPE.heimdall: 1,
        constants.CL_TYPE.heimdall_v2: 2,
    }
    heimdall_version = heimdall_version_map.get(l2_context.devnet_cl_type)
```

you'll probably need to import the `constants` module also.

```star
constants = import_module("../package_io/constants.star")
```